src/shared/components/locales-dropdown/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { injectIntl } from 'react-intl';
```

Comment:
Should the name be different than dropdown?

Reply:
oh wait, this probably is for mobile, right?

Reply:
Yes, you're right. This dropdown will be used on mobile devices.